RELEASE_NOTES

```diff
@@ -3,9 +3,75 @@ For more detailed information, please see the git log.
st 1.10.4, Boost.Python 1.71.0, barrnap 0.9, breseq 0.35.0, bx python, CDO 1.9.8, CNT ILP, CRPropa 3.1.5, CSBDeep 0.4.1, CSBDeep 0.4.1, Clang 9.0.1, Clang 9.0.1, ConnectomeWorkbench 1.3.2, CrossMap 0.3.9, cairo 1.16.0, cuDNN 7.6.4.38, cytoolz 0.10.1, DCMTK 3.6.5, deepTools 3.3.1, EMBOSS 6.6.0, ESMF 8.0.0, EasyBuild 4.1.0, ecCodes 2.15.0, FFTW 3.3.8, FLTK 1.3.5, FSL 6.0.2, FSL 6.0.2, FSL 6.0.3, Faber 0.3, Fiji 20170530, Fiji 20191119, fastp 0.20.0, fastp 0.20.0, freeglut 3.2.1, GARLI 2.01, GDAL 3.0.0, GDAL 3.0.2, GEOS 3.7.2, GEOS 3.8.0, GLibmm 2.49.7, GLibmm 2.49.7, GROMACS 2019.3, GROMACS 2019.3, GROMACS 2019.4, GSL 2.6, GSL 2.6, GTS 0.7.6, Ghostscript 9.50, GitPython 3.0.3, Globus CLI, GtkSourceView 3.24.11, GtkSourceView 4.4.0, Guile 1.8.8, Gurobi 9.0.0, Gurobi 9.0.0, Gurobi 9.0.0, g2clib 1.6.0, g2clib 1.6.0, g2lib 3.1.0, g2lib 3.1.0, gc 7.6.12, gettext 0.20.1, gffread 0.11.6, glibc 2.30, gradunwarp 1.1.0, gradunwarp 1.1.0, gsettings desktop, HDF 4.2.14, HDF5 1.10.5, HPL 2.3, Horovod 0.18.2, HyPhy 2.5.1, horton 2.1.1, horton 2.1.1, hwloc 2.1.0, ICU 64.2, ICU 65.1, IGMPlot 2.4.2, IGMPlot 2.4.2, ITSTool 2.0.6, ImageMagick 7.0.9, iccifort 2020.0.166, iimpi 2020.00, imkl 2020.0.166, impi 2019.6.166, intel 2020.00, Java 1.8.0_231, Jellyfish 2.3.0, Julia 1.3.1, jemalloc 5.2.1, KMC 3.1.1, LEMON 1.3.1, LibTIFF 4.1.0, Libint 2.0.3, Libint 2.0.3, LittleCMS 2.9, libcerf 1.13, libgd 2.2.5, libgeotiff 1.5.1, libiconv 1.16, libiconv 1.16, libmatheval 1.1.11, libpciaccess 0.16, libsigc++ 2.10.2, libsndfile 1.0.28, libtool 2.4.6, libunistring 0.9.10, libxc 2.2.2, libxc 2.2.2, libxml++ 2.40.1, libxml++ 2.40.1, libxml2 python, libxml2 2.9.10, libxslt 1.1.34, lpsolve 5.5.2.5, lxml 4.4.2, MACS2 2.2.5, MDTraj 1.9.3, MRtrix 3.0, MRtrix 3.0, Meld 3.20.1, Mesquite 2.3.0, Mothur 1.43.0, make 4.2.1, makedepend 1.0.6, mayavi 4.7.1, mhcflurry 1.2.4, mhcflurry 1.2.4, molmod 1.4.5, NCIPLOT 4.0, NCL 6.6.2, NCL 6.6.2, NLopt 2.6.1, ncl 2.1.18, ncview 2.1.7, netCDF C++4, netCDF Fortran, netCDF 4.7.1, netCDF 4.7.1, netCDF 4.7.1, networkx 2.4, numactl 2.0.13, ORCA 4.2.1, OpenFOAM Extend, OpenFOAM 6, OpenFOAM 7, OpenFOAM v1912, OpenMM 7.4.1, OpenMM 7.4.1, OpenMPI 4.0.2, OpenSlide 3.4.1, openslide python, ownCloud 2.5.4, PLUMED 2.5.3, PROJ 6.2.1, ParMETIS 4.0.3, ParMGridGen 1.0, Pillow SIMD, PyCairo 1.18.0, PyCharm 2019.3.1, PyGObject 3.34.0, pixman 0.38.4, plotly.py 4.4.1, pocl 1.4, pocl 1.4, prokka 1.14.5, pyBigWig 0.3.17, pydicom 1.2.2, pythran 0.9.4.post1, QUAST 5.0.2, Qualimap 2.2.1, QuickFF 2.2.4, Qwt 6.1.4, Qwt 6.1.4, R 3.6.2, R 3.6.2, ReFrame 2.20, SAMtools 1.10, SCOTCH 6.0.9, SUNDIALS 5.1.0, SWIG 4.0.1, Salmon 1.0.0, Sambamba 0.7.1, SuiteSparse 5.6.0, scikit learn, seqtk 1.3, snakemake 5.7.1, TinyDB 3.15.2, TreeShrink 1.3.2, tbl2asn 25.8, tcsh 6.22.02, tcsh 6.22.02, texinfo 6.7, time 1.9, torchvision 0.4.2, UDUNITS 2.2.26, WPS 4.0.1, WPS 4.0.2, WPS 4.1, WRF 4.0.1, WRF 4.0.2, WRF 4.1.3, worker 1.6.11, XZ 5.2.4, xorg macros, xprop 1.2.4, xproto 7.0.31, YAXT 0.6.2, yaff 1.6.0
```

Comment:
@migueldiascosta cleaned this up a bit in https://github.com/migueldiascosta/easybuild-easyconfigs/pull/29, together with other minor tweaks